Src/WitsmlExplorer.Api/Query/WbGeometryQueries.cs

```diff
@@ -12,11 +15,55 @@ public static WitsmlWbGeometrys GetWitsmlWbGeometryByWellbore(string wellUid, st
                 WbGeometrys = new WitsmlWbGeometry
                 {
                     Uid = "",
-                    UidWell = wellUid,
-                    UidWellbore = wellboreUid,
+                    WellUid = wellUid,
+                    WellboreUid = wellboreUid,
                     CommonData = new WitsmlCommonData()
                 }.AsSingletonList()
             };
         }
+
+
+
+        public static WitsmlWbGeometrys QueryById(string wellUid, string wellboreUid, string wbGeometryUid)
```

Comment:
maybe change the name of the function to something more specific? For example GetWitsmlWbGeometryById